conf/application.conf

```diff
@@ -375,10 +375,10 @@ db {
 }
 
 rating.stream {
-  baseUri = "http://localhost:3020"
+  baseUrl = "http://localhost:3020"
   method = "ratings"
 }
 
 redis {
-  ratingsSortedList = "ratingsAvg2"
+  ratingsSortedList = "ratingsSortedList"
```

Comment:
what does means "ratingsSortedList" , a new filter ?

Reply:
"ratingsSortedList" is actually a data structure of type SortedSet in Redis, see: https://redis.io/topics/data-types. Basically what I am doing is I dynamically calculate the average rating of a movie as entries come from the stream and update the value stored in Redis on the fly. 

Redis will create a data structure if none exist with that name automatically, being more like a nosql db.  Let me know if this is enough clarification.